testing.py

```diff
@@ -261,8 +261,8 @@ def test_base_lookups(self):
 
         # Needs mocking to prepare NINJA_BASES properly in parse.py
         with requests_mock.Mocker() as mock:
-            mock.get("https://poe.ninja/api/data/itemoverview?league=Metamorph&type=BaseType&language=en", json=data)
-            parse.NINJA_BASES = parse.get_ninja_bases()
+            mock.get("https://poe.ninja/api/data/itemoverview?league=Standard&type=BaseType&language=en", json=data)
```

Comment:
Why exactly is this bugging on Windows? Is changing it to Standard really the fix that we need?

Reply:
Its not bugging, I changed the get_ninja_bases to take in the league in config. For the tests Standard made more sense since Metamorph has a end date

Reply:
That information should be included in the PR description, or the commit description itself. The commit including this change basically just says that it's bugging on windows, so we changed the test so it doesn't bug.

Reply:
If you want pages describing what was changed and why for 5 lines of code I will never get any work done.  It is not that hard to understand the changes here